freqtrade/freqtradebot.py

```diff
@@ -648,7 +648,17 @@ def handle_stoploss_on_exchange(self, trade: Trade) -> bool:
                 trade.update(order)
                 result = True
             else:
-                result = False
+                # check if trailing stoploss is enabled and stoploss value has changed
+                # in which case we cancel stoploss order and put another one with new
+                # value immediately
+                if self.config.get('trailing_stop', False) and trade.stop_loss > order['price']:
```

Comment:
I think you need to be very careful to avoid being banned.
create / cancel / update order commands normally have a higher "cost" on exchange api's, so updating this once every 5 seconds (assuming default config) might be a good way to get your IP / account blocked

I would not do this on each iteration but only every X...

Reply:
what about adding a column in persistence which keeps update time of last adjustment ? then having a parameter in order_types: trailing_stoploss_interval ...

Reply:
yes (kinda) - but i'm not certain about order_types ... making that dict too complicated will make it difficult to configure ... at least it should work if the setting is not there - with a "sensible" default (whatever that is)